sdk/identity/identity/managed-identity-matrix.json

```diff
@@ -0,0 +1,19 @@
+{
+  "displayNames": {
+    "**/test-results.xml": ""
+  },
+  "include": [
+    {
+      "Agent": {
+        "msi_image": {
+          "OSVmImage": "env:LINUXNEXTVMIMAGE",
+          "Pool": "env:LINUXNEXTPOOL"
+        }
+      },
+      "ArmTemplateParameters": "@{deployMIResources = $true}",
```

Comment:
nit: should this be deployMsiResources or should we update line 8 to say "mi_image" instead of "msi_image" ? Just want to be consistent on one acronym

Reply:
I think since "MSI" is going to be obsolete soon, we should change it to MI. But I'll do that in a followup PR just for those updates.